### PR TITLE
[ADD] Mail integration methods

### PR DESCRIPTION
**Pull request by aitorbouzas**
Three new methods:

- `message_new` parses the mail and creates default values for the ticket
- `message_update` subscribes partners discussing in the thread when a new mail is received
- `message_get_suggested_recipients` checks whether the ticket has partner, if so it suggests to add it as a follower (if it is not added already). If the ticket hasn't a partner, it will check if it has a `partner_email` and add it to suggested recipients.

### CODE REVIEW

**Comment by aitorbouzas** [comment]
@emagdalenaC2i @carlosDomatix

Could you please review?

File: helpdesk/models/helpdesk_ticket.py [hunk 1]
```diff
@@ -135,3 +136,69 @@ def _track_template(self, tracking):
                                {"composition_mode": "mass_mail"})
 
         return res
+
+    # ADDING
```
Comment (emagdalenaC2i) [inline helpdesk/models/helpdesk_ticket.py]: Delete comment

**Review by emagdalenaC2i — APPROVED** [review]
The code seems fine, but I could not test it

**Comment by aitorbouzas** [comment]
@emagdalenaC2i Thanks, testing should be done locally with a incoming mail server correctly configured to create new tickets (or alias).

**Comment by carlos-domatix** [comment]
LGTM

**Comment by aitorbouzas** [comment]
@carlosDomatix thank you
Waiting for @agenterp

**Pull request merged**